### ISSUE TITLE
`SubViewport` children of `SubViewportContainer` don't all receive mouse click input events due to being set to handled

### ISSUE THREAD

**Issue by razcore-rad** [godotengine/godot#89073]
### Tested versions

- Reproducible in: 4.2.2.rc1.official.c7fb0645a

### System information

Ubuntu 22.10, Vulkan, NVIDIA GeForce RTX™ 2060 / NVIDIA GeForce RTX™ 2060

### Issue description

Having a setup like:
```
- SubViewportContainer
  - SubViewport1
  - SubViewport2
  - ...
```
Only one `SubViewport` gets a mouse button pressed event (possibly other mouse events too). Probably because `SubViewportContainer` sets `SubViewport.handle_input_locally` to `false` and because the container itself sets the event as handled.

I tested the setup with an `Area2D._input_event()`. It might have to do with `Viewport.physics_object_picking` being set to `true` so that `Area2D` can interact with the mouse.

### Steps to reproduce

Open the Minimal Reproduction Project. Make sure to turn on *Debug > Vissible Collision Shapes* and click on the `Area2D`s. Notice that only one print function is called, but the message should be printed twice.

The hack solution is to force `SubViewport.handle_input_locally` to `true`.

### Minimal reproduction project (MRP)

[godot-dbg.zip](https://github.com/godotengine/godot/files/14468200/godot-dbg.zip)

**Comment by Sauermann** [comment]
I can confirm, that this is the current behavior. 
I don't see an easy way of how to implement this behavior into the engine.

Can you please open a proposal, so that we can get feedback from the community about the need for this functionality:
https://github.com/godotengine/godot-proposals/

A workaround would be to use two `Area2D` instead of a single `SubViewportContainer`. You can find a demo-project of how to achieve this in https://github.com/godotengine/godot/issues/79128#issuecomment-1624813289

**Comment by razcore-rad** [comment]
Indeed, thanks for pointing out the `Area2D` demo. I have two layouts, a ghost layout with one `SubViewport` on top of another using `SubViewportContainer`s and it works. The other, with a split view where I want to send input events from one `SubViewport` to the other wasn't working no matter what I tried.

I managed to make it work with the `Area2D` setup isntead.

https://github.com/godotengine/godot/assets/1177508/26aa7ad3-4441-422f-b9e8-4a87c3f4e834

**Comment by Sauermann** [comment]
@razcore-rad Thank you for the report. It helped me to understand the intention of `Viewport.handle_input_locally` better.
In #77926 I have implemented a method for achieving the use-case described in this issue-report.

An alternative workaround might be to set `SubViewport.handle_input_locally = true` in the SubViewports.
